app/grants/templates/grants/landing/index.html

```diff
@@ -45,6 +45,17 @@
       align-items: center;
     }
 
+    .active-clr-banner {
+      height: 111px !important;
+    }
+
+    @media (max-width: 991.98px) {
```

Comment:
Could we add a utility to manage responsive fonts rather than handling them inline?